app/uk/gov/hmrc/tariffclassificationfrontend/models/SampleStatus.scala

```diff
@@ -18,11 +18,12 @@ package uk.gov.hmrc.tariffclassificationfrontend.models
 
 object SampleStatus extends Enumeration {
   type SampleStatus = Value
-  val AWAITING, MOVED_TO_ACT, MOVED_TO_ELM, SENT_FOR_ANALYSIS, SENT_TO_APPEALS, STORAGE, RETURNED_APPLICANT,
+  val NONE, AWAITING, MOVED_TO_ACT, MOVED_TO_ELM, SENT_FOR_ANALYSIS, SENT_TO_APPEALS, STORAGE, RETURNED_APPLICANT,
   RETURNED_PORT_OFFICER, RETURNED_COURIER, DESTROYED = Value
 
   def format(status: Option[SampleStatus], initialCaps: Boolean = true): String = {
     val text = status match {
+      case Some(NONE) => "none"
```

Comment:
You can have this as the `_` case

Reply:
true

Reply:
as in, you can remove this line or I think do `case _ | "none"`